enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: Object;
```

Comment:
The first statement sounds reasonable, but the second may be not. `Storage` originally is an private class lived in `CachedInputFileSystem.js`. Because class `CachedInputFileSystem` has some member propertis which are `Storage` type. That's why i expose it as a separate class. It has not thing to do with `Tabpable`. At least, with some breakpoints, no evidence found.